content/en/docs/reference/_index.md

```diff
@@ -35,6 +35,9 @@ client libraries:
 
 - [Kubernetes Go client library](https://github.com/kubernetes/client-go/)
 - [Kubernetes Python client library](https://github.com/kubernetes-client/python)
+- [Kubernetes Java client library](https://github.com/kubernetes-client/java)
+- [Kubernetes dotnet client library](https://github.com/kubernetes-client/csharp)
```

Comment:
This is still WIP. Maybe we should list it here after it graduates.

Reply:
@xiangpengzhao I took the list from https://www.terraform.io/docs/providers/helm/index.html#example-usage

Maybe I should add a beta column there too?

But I will remove it.

Reply:
/cc @chenopis